custom_components/luxtronik/__init__.py

```diff
@@ -145,6 +147,14 @@ def setup_internal(hass, data, conf):
         manufacturer=get_manufacturer_by_model(model),
         model=model,
     )
+    
+    MK1 = luxtronik.get_value('parameters.ID_Einst_MK1Typ_akt')
+    MK2 = luxtronik.get_value('parameters.ID_Einst_MK2Typ_akt')
+    MK3 = luxtronik.get_value('parameters.ID_Einst_MK3Typ_akt')
+    CoolingPresent = ((MK1 in [3,4]) | (MK2 in [3,4]) | (MK3 in [4]))
```

Comment:
Magic numbers, I think you should define constants for these, or better, add support for these to the python-luxtronik library.

Reply:
Also, create helper methods for this because this pattern is repeated for the detection of the setpoint parameter.

Reply:
Detection functions created in luxtronik_device.py